contracts/VestedPayment.sol

```diff
@@ -61,7 +86,13 @@ contract VestedPayment is Ownable {
     }
   }
 
-  // claim the tokens, they can be claimed only by the owner of the contract
+  //
+  /**
+     @dev Claim the tokens, they can be claimed only by the owner
+     of the contract
+
+     @param amount how much tokens to be claimed
```

Comment:
s/how much/how many

(it's how many because it's about a quantity (http://www.grammar.cl/english/how-much-how-many.htm)